Duplicated test in src/lib/dateUtils.test.js

Found via ESLint and local eslint-plugin-jest tinkering:

```
/Volumes/Dev/github/mozilla/experimenter/app/experimenter/nimbus-ui/src/lib/dateUtils.test.ts
  41:10  error  Describe block title is used multiple times in the same describe block  jest/no-identical-title
```

https://github.com/mozilla/experimenter/blob/2734dc2cd424497218c1bb25f25de3f5054bfe46/app/experimenter/nimbus-ui/src/lib/dateUtils.test.ts#L35-L45

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-1523)

--- a/app/experimenter/nimbus-ui/src/lib/dateUtils.test.ts
+++ b/app/experimenter/nimbus-ui/src/lib/dateUtils.test.ts
@@ -9,7 +9,7 @@ import {
 } from "./dateUtils";
 import { mockSingleDirectoryExperiment as expFactory } from "./mocks";
 
-const FAKE_DATE = "Thu Dec 12 2020";
+const FAKE_DATE = "Sat Dec 12 2020";
 const FAKE_ISO_DATE = "2020-12-25T15:28:01.821657+00:00";
 
 describe("humanDate", () => {
@@ -29,12 +29,6 @@ describe("humanDate", () => {
     expect(humanDate("2021-06-21T10:00:00Z")).toEqual("Jun 21");
     expect(humanDate("2021-06-21T00:00:00Z")).toEqual("Jun 21");
     expect(humanDate("2021-06-21")).toEqual("Jun 21");
-  });
-});
-
-describe("addDaysToDate", () => {
-  it("should add days to a date and return a date string", () => {
-    expect(addDaysToDate(FAKE_DATE, 2)).toEqual("Mon Dec 14 2020");
   });
 });
 

